src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -337,4 +337,80 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
 usage: NuGet locals &lt;all | http-cache | global-packages | temp&gt; [--clear | -c | --list | -l]
 For more information, visit http://docs.nuget.org/docs/reference/command-line-reference</value>
   </data>
+  <data name="Error_MSBuildUnableToOpenProject" xml:space="preserve">
+    <value>MSBuild was unable to open Project '{0}'.</value>
+    <comment>{0} - Project/csproj path</comment>
+  </data>
+  <data name="Error_DotnetNotFound" xml:space="preserve">
+    <value>Failed to locate dotnet in dotnet add pkg command.</value>
+  </data>
+  <data name="Error_FailedToCreateRandomFile" xml:space="preserve">
+    <value>Failed to create random file for dotnet add pkg command.</value>
```

Comment:
I would not know what to do with this message as a user. Write out the path instead.